resume/src/components/Basics.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import Profiles from './Profiles'
+import '../App.css';
+
+function Basics(props){
+    let basics = props.data
+    return(
+        <div className="basics">
+            <img src={basics.picture}></img>
+            <h1>{basics.name}</h1>
+            <small>{basics.label}</small>
+            <hr/>
+            <p className='contact'>{basics.email}</p>
+            <p className='contact'>{basics.phone}</p>
+            <p className='contact'>{basics.website}</p>
+            <hr/>
```

Comment:
I believe HRs are meant to be used semantically as a break instead of for visually, according to MDN: "Historically, this has been presented as a horizontal rule or line. While it may still be displayed as a horizontal rule in visual browsers, this element is now defined in semantic terms, rather than presentational terms, so if you wish to draw a horizontal line, you should do so using appropriate CSS." So maybe use the HRs more selectively, or use CSS to achieve the same effect!